Sources/iOS/Request+Trigger.swift

```diff
@@ -6,6 +6,8 @@
 //
 //
 
+import UIKit
```

Comment:
Since this is in the iOS code folder the header is ok, but do we actually need it?

Reply:
I'll change it to `import UIKit.UIDevice`